select2/widgets.py

```diff
@@ -10,6 +10,11 @@
 except ImportError:
     from django.forms.util import flatatt
 from django.utils.datastructures import MultiValueDict
+try:
+        from django.utils.datastructures import MergeDict
```

Comment:
Whoops. Good catch. I copy/pasted from github and I caught it the first go around, but missed it this time.  Fixed.